src/obstacle_avoidance/src/OlafNode.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * Created By: Gareth Ellis
+ * Created On: July 16th, 2016
+ * Modified By: Marinah Zhao
+ * Modified On: Feb 12, 2018
+ * Description: A node that subscribes to a lidar topic publishing twist messages for the robot to move.
+ */
+
+#include <OlafNode.h>
+
+OlafClass::OlafClass(int argc, char **argv, std::string node_name) {
```

Comment:
I think we could change this to be a bit more descriptive of the class's functionality. Maybe something like `ConeAvoider`. In general, we try to avoid putting the type of something in it's name (ex. `double my_value` instead of `double my_double_value`).

Reply:
Done.